macos/FluentUIResources-macos/Strings/es-MX.lproj/FluentUI.strings

```diff
@@ -0,0 +1,16 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+/* Accessibility label for the date picker view */
+"DATEPICKER_ACCESSIBILITY_DATEPICKER_LABEL" = "Date picker";
```

Comment:
These haven't yet been localized. The translations should be updated in a future call to tdbuild now that the strings are registered.